nodes/object/object_bounding_box.py

```diff
@@ -1,21 +1,31 @@
 import bpy
+from bpy.props import *
+from ... events import executionCodeChanged
 from mathutils import Vector
 from ... base_types.node import AnimationNode
 
 class ObjectBoundingBoxNode(bpy.types.Node, AnimationNode):
     bl_idname = "an_ObjectBoundingBoxNode"
     bl_label = "Object Bounding Box"
-
+    
+    useWorldSpace = BoolProperty(name = "Use World Space", default = True,
+        update = executionCodeChanged)
+    
     def create(self):
         self.inputs.new("an_ObjectSocket", "Object", "object").defaultDrawType = "PROPERTY_ONLY"
         self.outputs.new("an_VectorListSocket", "Vertices", "vertices")
         self.outputs.new("an_EdgeIndicesListSocket", "Edges", "edges")
         self.outputs.new("an_PolygonIndicesListSocket", "Polygons", "polygons")
-
+        
+    def drawAdvanced(self, layout):
+        layout.prop(self, "useWorldSpace")
+    
     def execute(self, object):
         if object is not None:
```

Comment:
please change that to:
if object is None: return [], [], []
(i know that it was my 'mistake', not yours)